screens/Messages/style.js

```diff
@@ -103,6 +102,26 @@ const styles = StyleSheet.create({
     alignSelf: 'flex-start',
     fontSize: 15,
   },
+  leftArrow: {
+    position: 'absolute',
+    backgroundColor: '#E8E8E8',
+    width: 20,
+    height: 30,
+    bottom: 0,
+    borderBottomRightRadius: 25,
+    left: 20,
+    zIndex: -1,
+  },
+  rightArrow: {
+    position: 'absolute',
+    backgroundColor: '#5DB075',
+    width: 20,
+    height: 30,
+    bottom: 0,
+    borderBottomLeftRadius: 25,
+    right: 20,
+    zIndex: -1,
+  },
```

Comment:
if 90% of the styles are the same as in your case for `leftArrow` and `rightArrow`, it's better to have one common style(e.g. `arrow`) and then in `leftArrow` and `rightArrow` store the differences. Please update

Reply:
Updated as mentioned.